extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -409,6 +422,39 @@ export class AmpAdExit extends AMP.BaseElement {
     this.init3pResponseListener_();
   }
 
+  /**
+   * Determine if `attribution-reporting` is supported by user-agent. Should only return
+   * true for Chrome 92+.
+   * @visibleForTesting
+   * @return {boolean}
+   */
+  detectAttributionReportingSupport() {
+    return this.win.document.featurePolicy
+      ?.features()
+      .includes('attribution-reporting');
+  }
+
+  /**
+   * Extracts the keys from the `browserAdConversion` data creates a
+   * string to be used as the `features` param for the `window.open()` call.
+   * @param {JsonObject} adConversionData
+   * @return {?string}
+   */
+  getAttributionReportingValues_(adConversionData) {
+    if (!adConversionData) {
+      return;
+    }
+
+    const parts = [];
```

Comment:
How about defaulting to `const parts = ['noopener']`? It shouldn't hurt?

Reply:
I think the only difference would be if the config looks like this
```json
{
  "targets": {
    "landingPage": {
      "finalUrl": "https://example.com",
      "behaviors": {
        "browserAdConversion": {}
      }
    }
  }
}
```
Which would lead to `window.open('example.com', '_blank')` vs `window.open('example.com', '_blank', 'noopener')`. I learned that older browsers will do weird things if they get the optional third param but that shouldn't apply here as this path should only be Chrome 92+. 

Just in case I added an early exit if no keys in the object and moved 'noopener' to where you suggest.